portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,12 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+.link {
+    text-decoration: none;
```

Comment:
Make sure your indentation is consistent with above (namely two spaces rather than four) - https://google.github.io/styleguide/htmlcssguide.html#Indentation (holds for CSS too, although the example is in HTML)

Reply:
Done!